pyodide_build/pypabuild.py

```diff
@@ -242,6 +242,23 @@ def make_command_wrapper_symlinks(symlink_dir: Path) -> dict[str, str]:
     return env
 
 
+@contextmanager
+def _get_symlink_dir(env: dict[str, str], build_dir: Path | None):
```

Comment:
Could you please check if it works well with `--continue` flag? I think it will, but just in case.

Reply:
I tried using `--continue` on failures in a few different places and it seems to work.